eng/pipelines/coreclr/jitstress.yml

```diff
@@ -1,18 +1,18 @@
 trigger: none
 
 schedules:
-- cron: "0 4 * * 1,3,5"
-  displayName: Mon, Wed, Fri at 8:00 PM (UTC-8:00)
+- cron: "0 4 * * *"
+  displayName: Daily at 8:00 PM (UTC-8:00)
   branches:
     include:
     - master
```

Comment:
Master no longer exists, I'd revert these lines of the change as this will only after the branch you check it in.

Reply:
I suppose we could just delete the "master" section of this (and other) yml files, since they're unused in the release branches anyway.

I did this just as a "cherry-pick" from the 'main' branch change, so it doesn't really hurt, either